PrebidMobile/PrebidMobileRendering/Prebid/Integrations/MediationAPI/PrebidMediationDelegate.swift

```diff
@@ -0,0 +1,24 @@
+/*   Copyright 2018-2021 Prebid.org, Inc.
+ 
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+ 
+  http://www.apache.org/licenses/LICENSE-2.0
+ 
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+  */
+
+import Foundation
+
+@objc
+public protocol PrebidMediationDelegate {
```

Comment:
This protocol might be used by publishers in order to create their own delegates. Hence, we should annotate it as precisely as possible. Please, add respective comments to the protocol itself and its methods.